code/app/lagrange.py

```diff
@@ -3,66 +3,61 @@
 import arrayfire as af
 
 from utils import utils
-from scipy import special as sp
 
 from app import global_variables as gvar
 
 def LGL_points(N):
 	"""
-	Returns the N Legendre-Gauss-Laguere points which are
+	Returns the :math: `N` Legendre-Gauss-Laguere points which are
 	the roots of the equation
 	:math::
-		#(1 - x^{2})L'_{N} = 0
+		(1 - x^{2})L'_{N} = 0
```

Comment:
Curly braces are needed only when we have to make more than one
letter as subscript/superscript. Here, you don't need curly braces for 2 and N.